charts/trento-server/values.yaml

```diff
@@ -68,5 +72,21 @@ rabbitmq:
   auth:
     username: trento
     password: trento
+    tls:
+      enabled: "{{ .Values.global.rabbitmq.mtls.enabled }}"
+      autogenerated: false
+      existingSecret: "{{ .Values.global.rabbitmq.mtls.existingServerSecret }}"
+  extraConfiguration: |
+    {{- if .Values.global.rabbitmq.mtls.enabled }}
+    listeners.ssl.default = {{ .Values.global.rabbitmq.servicePort }}
+    ssl_options.cacertfile = /opt/bitnami/rabbitmq/certs/ca_certificate.pem
+    ssl_options.certfile = /opt/bitnami/rabbitmq/certs/server_certificate.pem
+    ssl_options.keyfile = /opt/bitnami/rabbitmq/certs/server_key.pem
+    ssl_options.verify = verify_peer
+    ssl_options.fail_if_no_peer_cert = true
```

Comment:
should we add `listeners.tcp = none` at this point, to completely disable the unsecured listener?

Reply:
Yes, done in https://github.com/trento-project/helm-charts/pull/132/commits/a6200ba0e200e78c0e75101d578c51bcd106c14b.

(I verified that the amqp listener would otherwise be enabled by default)